packages/bpk-component-breadcrumb/src/BpkBreadcrumb.js

```diff
@@ -26,24 +26,66 @@ import STYLES from './BpkBreadcrumb.scss';
 
 const getClassName = cssModules(STYLES);
 
+type SchemaMetaDataItem = {
+  url: string,
+  label: string,
+};
+
 export type Props = {
   children: Node,
+  schemaMetaData: ?(SchemaMetaDataItem[]),
   label: string,
 };
 
+const buildMetaData = (schemaMetaData: SchemaMetaDataItem[]): string => {
```

Comment:
I wonder if it's worth adding a link to the schema in a comment above this function, as I can imagine somebody unfamiliar with it coming along in the future and not knowing why the data needs to be formatted this way.

Reply:
I've added a link to the google structured data reference above